src/commands/synthetics/run-tests-command.ts

```diff
@@ -195,10 +198,22 @@ export class RunTestsCommand extends Command {
   private async resolveConfig() {
     // Defaults < file < ENV < CLI
 
+    let overrideConfigPath = this.config.configPath
```

Comment:
I think this will never be defined, since the config file can't specify the path to... itself. I believe we want to start with the env config as the baseline, then put the CLI on top of it.